src/types.ts

```diff
@@ -240,6 +240,10 @@ export interface Instance {
      */
     uploadMedia(media: Buffer): Promise<string>;
 
+    startUrlUploadJob(urls: string[]): Promise<UrlToJobId[]> | InvalidInputMessage; // returns job ids
+
+    checkBulkUploadJob(jobIds: string[]): Promise<UploadJobStatus[]> | InvalidInputMessage; // return status of the jobs
```

Comment:
will we ever care about having a singular `checkUploadJob` to find the status of a specific job?

Reply:
you could always pass a list of just one element to this method but yeah I'll write that method too